GameData/RealismOverhaul/RO_SuggestedMods/RealScaleBoosters/RO_RSB_Ariane.cfg

```diff
@@ -268,34 +270,29 @@
     @manufacturer = Airbus Defence and Space
     @description = The second stage of the Ariane 5 launch vehicle family (ESC-A variant). Contains guidance capability.
 
-    @mass = 4.375
+    @mass = 4.45
     @crashTolerance = 14
     @breakingForce = 250
     @breakingTorque = 250
     @maxTemp = 773.15
     %skinMaxTemp = 873.15
 
-    @MODULE[ModuleRCS]
+    //  56 N Gas Hydrogen (GH2) attitude control thrusters for each axis (using liquid hydrogen directly for now).
```

Comment:
Yep! From the Ariane 5 User's Manual (page 204):

```
During the powered flight, the attitude control in pitch and yaw is ensured by the gimballing of the nozzle, and 4 GH2 thrusters are used for roll control. During the ballistic phase, roll, pitch and yaw control uses 4 clusters of 3 GH2 thrusters. 2 GO2 thrusters are also implemented for longitudinal boosts.
```

So even now it is not accurate enough (we cannot use different propellants for different RCS transforms on the same part).